spec/recipes/cask_spec.rb

```diff
@@ -1,21 +0,0 @@
-require 'spec_helper'
-
-describe 'homebrew::cask' do
-  context 'default user' do
-    let(:chef_run) do
-      ChefSpec::SoloRunner.new.converge(described_recipe)
-    end
-
-    before(:each) do
-      allow(Homebrew).to receive(:owner).and_return('vagrant')
-    end
-
-    it 'manages the Cask Cache directory' do
-      allow(Dir).to receive(:exist?).with('/Library/Caches/Homebrew').and_return(true)
-      expect(chef_run).to create_directory('/Library/Caches/Homebrew/Casks').with(
-        user: 'vagrant',
-        mode: '0775'
-      )
-    end
-  end
-end
```

Comment:
What was the error here that needed us to delete the tests?

Reply:
too many here is a gist of running rspec after only modifying the library to point to the new name for the class https://gist.github.com/Stromweld/2d254b6308db6e447264110168dfed9a